composer.json

```diff
@@ -19,7 +19,8 @@
         "composer/installers": "^1.0.20",
         "drupal/core": "8.0.*",
         "drush/drush": "dev-master",
-        "drupal/console": "dev-master"
+        "drupal/console": "dev-master",
+        "drupal-composer/drupal-scaffold": "dev-master"
```

Comment:
Nitpick, but since this is one of the base "drupal-composer" tools, could we move it up the list ?
The bottom of that list os probably going to be where each project adds its own contrib modules :-)